packages/@aws-cdk/aws-codedeploy/README.md

```diff
@@ -1,12 +1,15 @@
 ## The CDK Construct Library for AWS CodeDeploy
 
-### Applications
+AWS CodeDeploy is a deployment service that automates application deployments to Amazon EC2 instances, on-premises instances, serverless Lambda functions, or Amazon ECS services.
+
+The CDK currently supports Amazon EC2, on-premise and AWS Lambda applications.
+
+### EC2/on-premise Applications
```

Comment:
You don't mention Lambda Applications in the ReadMe at all - probably worth at least a quick note ("Lambda Applications are very similar to EC2/on-premise Applications", for example).